app/Models/Shoutbox.php

```diff
@@ -38,7 +38,7 @@ class Shoutbox extends Model
 
     protected $guarded = [];
 
-    public function user(){
+    public function user() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->